Hush/Assets/Scripts/Doors/Door.cs

```diff
@@ -83,7 +83,7 @@ private void OpenDoor(InputAction.CallbackContext context)
 
         private void LoadNext()
         {
-            SceneManager.Instance.LoadNextScene();
+            SceneManager.Instance.LoadTempPowerScene();
```

Comment:
No need to load Power Scene, powers will be in the Scene UI already. Talk to @daniela-venuta